Softeq.NetKit.Chat.SignalR/Hubs/HubEvents.cs

```diff
@@ -10,6 +10,8 @@ public static class HubEvents
         public const string MessageUpdated = "MessageUpdated";
         public const string LastReadMessageChanged = "LastReadMessageChanged";
 
+        public const string SystemMessageAdded = "SystemMessageAdded";
```

Comment:
we don't need new notification type for system messages

Reply:
Done